inputs/dust/SI_strat_AB.in

```diff
@@ -0,0 +1,135 @@
+# ========================================================================================
```

Comment:
Only the two dust coagulation input files show be a part of this PR

Reply:
this input file is deleted in the new code